docs/content/commands/npm-outdated.md

```diff
@@ -97,6 +96,16 @@ When running `npm outdated` and `npm ls`, setting `--all` will show all
 outdated or installed packages, rather than only those directly depended
 upon by the current project.
 
+#### `in-range`
+
+* Default: null
+* Type: null or Boolean
+
+When running `npm outdated` setting `--in-range` will limit output to only
+those packages with existing versions in range of the current project's
+dependencies. `--no-in-range` will limit output to only those packages with
+updates that are out of range of the current project's dependencies.
+
```

Comment:
so with these options, how do i override `NPM_CONFIG_IN_RANGE=false npm outdated` (or true)? do i have to do `--in-range=null`?

Generally a boolean isn't a great choice when there's three options, so perhaps an enum would be clearer here?

Reply:
@ljharb see the discussion on the if statements above, it is as you say checking for the default `null` vs an explicit true or false.  I agree it's not ideal and that discussion is why this PR went back to draft status to be discussed and solved as part of a better solution that lets `npm update --dry-run` solve the "What can be updated problem" first.